### PR TITLE
updated table and queries

### PR DESCRIPTION
**Pull request by AhmedNazirMusah**
On this milestone;
- Added a column species of type string to the animals table and modified my schema.sql file.
- Added all the required data to the table and add these insertions to the data.sql file.
- The following is the list of all transactions with their respective screenshots. All these transactions were saved in the mytransactions.sql file.

First Transaction. 
TRANSCTION FOR SETTING species to unspecified

Second Transaction.
update species to digimon
update the rest of the species to pokemon

Third Transaction.
delete all records and rollback

Last transaction.
first multiply and roll back
last commit

- Added all Queries to the queries.sql file
The image below highlights all the queries added.
result for all queries

### CODE REVIEW

File: mytransactions.sql [hunk 1]
```diff
@@ -0,0 +1,30 @@
+BEGIN;
+UPDATE animals
+SET species = 'unspecified';
+ROLLBACK;
```
Comment (omar-muhamad) [inline mytransactions.sql:4]: - [ ] As per this [requirement](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_query_update_tables.md#:~:text=Verify%20that%20change%20was%20made.%20Then%20roll%20back%20the%20change%20and%20verify%20that%20the%20species%20columns%20went%20back%20to%20the%20state%20before%20the%20transaction.) Please verify that change was made and after the rollback verify that the species columns went back to the state before the transaction.

File: mytransactions.sql [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+BEGIN;
+UPDATE animals 
+SET species = 'digimon'
+WHERE name LIKE '%mon'; 
+UPDATE animals 
+SET species = 'pokemon'
+WHERE species is NULL; 
+COMMIT;
```
Comment (omar-muhamad) [inline mytransactions.sql:13]: - [ ] As per this [requirement](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_query_update_tables.md#:~:text=Verify%20that%20change%20was%20made%20and%20persists%20after%20commit.), Kindly verify that change was made and persists after commit.

File: mytransactions.sql [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+BEGIN;
+DELETE FROM animals;
+ROLLBACK;
```
Comment (omar-muhamad) [inline mytransactions.sql:17]: - [ ] As per this [requirement](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_query_update_tables.md#:~:text=After%20the%20rollback%20verify%20if%20all%20records%20in%20the%20animals%20table%20still%20exists), Please verify if all records in the `animals` table still exist after the rollback

**Review by omar-muhamad — CHANGES_REQUESTED** [review]
# Changes Required ♻️
### Hi @AhmedNazirMusah,

Nice job, you are almost there.



## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- You used the correct workflow.  ✔
- All linters checks are ok. ✔

## (Changes Required) Aspects to improve: ♻️

- [ ] Please check the inline comments below.

## [Optional] suggestions:

- Nothing to mention.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: mytransactions.sql (hunk 1 above)
Comment (AhmedNazirMusah) [reply mytransactions.sql:4]: Thank you @Omar-Muhamad, this was verified and captured in the screenshot of the first transaction.

File: mytransactions.sql (hunk 2 above)
Comment (AhmedNazirMusah) [reply mytransactions.sql:13]: Also, this was verified and captured in the screenshot of the second transaction.

File: mytransactions.sql (hunk 3 above)
Comment (AhmedNazirMusah) [reply mytransactions.sql:17]: Hi, @Omar-Muhamad  this was also verified by the screenshot of the third transaction in the pull request.

**Review by 1GORDON — APPROVED** [review]
# Status: APPROVED 🟢 

Hi @AhmedNazirMusah ✋

Your project is complete!  There is nothing else to say other than... it's time to merge it: ship it:
Congratulations! :tada: :clap: :clap: :100:

## Highlights!
- Followed Gitflow ✔️
- Good commit messages ✔️
- Descriptive Pull request ✔️
- Professional readme ✔️

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@1GORDON` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**